engine/src/cython/initialize.cpp

```diff
@@ -522,9 +522,9 @@ ucp_ep_h CreateUcpEp(ucp_worker_h ucp_worker,
 * and the cache we use for receiving messages
 *
 */
-std::pair<std::pair<std::shared_ptr<CacheMachine>,std::shared_ptr<CacheMachine> >, int> initialize(int ralId,
+std::pair<std::pair<std::shared_ptr<CacheMachine>,std::shared_ptr<CacheMachine> >, int> initialize(uint16_t ralId,
 	std::string worker_id,
-	int gpuId,
+	int /*gpuId*/,
```

Comment:
if we are not using this parameter, which should just take it out

Reply:
Removed.